sysdig/internal/client/v2/model.go

```diff
@@ -913,7 +913,10 @@ type alertV2ChangeWrapper struct {
 }
 
 type CloudAccountCredentialsMonitor struct {
-	AccountId string `json:"accountId"`
+	AccountId   string `json:"accountId"`
+	RoleName    string `json:"roleName"`
+	SecretKey   string `json:"secretKey"`
+	AccessKeyId string `json:"accessKeyId"`
```

Comment:
Looking at the credentials object in monitor https://github.com/draios/backend/blob/main/sysdig-backend/legacy/src/main/java/com/draios/rest/api/model/cloudmonitoring/ProviderCredentialsV2.java
* `SecretKey` should be mapped to `json:key`
* `AccessKeyId` should be mapped to `json:id`